source/user-portal/assets/favRecipe.js

```diff
@@ -0,0 +1,108 @@
+class FavRecipe extends HTMLElement {
+    constructor() {
+      super();
+      this.attachShadow({mode: 'open'});
+    }
+  
+    set data(data) {
+        const styleElem = document.createElement('style');
+        const styles = `
+        
+            article{
+                display: grid;
+                align-items: center;
+                width: 20vw;
+                grid-template-rows: 110px 240px;
+                height: 20vw;
+                row-gap: 5px;
+                border: 3px solid black;
+                background-color: white;
+                border-radius: 10px;
+            }
+
+            a {
+                font-family: 'Varelia Round', sans-serif;
+                font-size: 2.5vh;
+                align-items: center;
+                margin-top: 20px;
+                margin-left: 15px;
+                margin-right: 15px;
+                max-width: 20vw;
+                overflow: hidden;
+            }
+          
+            p {
+                font-family: 'Varela Round', sans-serif;
+                font-size: 2.5vh;
+                align-items: center;
+                margin-top: 20px;
+                margin-left: 15px;
+                margin-right: 15px;
+            }
+        
+            img {
+                display: flex;
+                object-fit: cover;
+                max-height: 240px;
+                display: block;
+                margin-left: auto;
+                margin-right: auto;
+                margin-top: 0%;
+                width: 90%;
+            }
+
+            div {
+                width: 60px;
+                height: 60px;
+                background: #b90c0c;
+                border-radius: 50%;
+                position: relative; 
+                visibility: visible; 
+                left: 300px; 
+                bottom: 60px;
+                border-style: solid;
+                border-width: 2px;
+                border-color: black;
+            }
+
+            .recipe-time {
+                color: white;
+                margin-left:12px;
+                text-align: center;
+            }
+        `;
+        
+        styleElem.innerHTML = styles;
+
+        const recipe = document.createElement('article');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8429335)
```suggestion
        const recipe = document.createElement("article");
```